starter-code/src/StackDataStructure.js

```diff
@@ -1,2 +1,34 @@
 function StackDataStructure () {
+  var that = this;
+  this.stackControl = [];
+  this.MAX_SIZE = 8;
+  this.isEmpty = function () {
```

Comment:
Remember to use 
StackDataStructure.prototype.isEmpty = function () {
  ...
}
for implementing methods.